packages/runtime/container-runtime/src/summary/orderedClientElection.ts

```diff
@@ -416,13 +416,26 @@ export class OrderedClientElection
 			change = true;
 		}
 		if (change) {
+			this.logger.sendTelemetryEvent({
```

Comment:
In a session with 100 users, will we see 100 events in our telemetry when summarizer client exits?
If yes, anything we can do to reduce amount of data we generate (but still have enough data to make sense of what happens)?

Reply:
We could log only once on non-summarizer clients 
Thoughts ?

Reply:
Up to you how to structure it. You can observe that in service tests - as long as we are not putting too much data, we should be fine